usaspending_api/references/migrations/0059_auto_20170330_2107.py

```diff
@@ -12,9 +12,4 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.AlterField(
```

Comment:
Ohhhh.... rats.  I was hoping to slip some edits to recent migrations through while the edited migrations only existed on my own machine... but I don't think this qualifies for that anymore.

I'd better figure out how to do this as separate migrations instead.